about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Plug about</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <section class="start">
+        <header class="realHead">
+            <div class="navLine">
+                <nav>
+                    <div class="sideMback">
+                        <img class="sideMenu" onclick="openMobileMenu()" src="img/sideMenu.png" alt="">
+                    </div>
+                    <div class="navSlide">
+                        <div class="mediNbac">
+                            <ul class="mediaNav">
+                                <li><img src="img/facebook.png" alt=""></li>
+                                <li><img src="img/twitter.png" alt=""></li>
+                                <li>English</li>
+                                <li>My page</li>
+                                <li>Login</li>
+                            </ul>
+                        </div>
+                        <div class="relNav">
+                            <div>
+                                <a href="index.html"><ul class="Navle">
+                                    <li><img class="logoN" src="img/plug_icon.png" alt=""></li>
+                                    <li>
+                                        <h1>MyPlug</h1>
+                                    </li>
+                                </ul></a>
+                            </div>
```

Comment:
**Changes Suggested**
- [ ] There is too much use of the `div` element where we could actually use HTML5 `article` element. Let us try and follow this form when organizing our elements everywhere in our code base. `<section><article>{p, ul, div etc}</article></section>`. The use of HTML5 semantics increases the accessibility of our web pages.

Reply:
Am not sure about this because in html/css best practice it is not mentioned anywhere.